AsciiPixel/src/AsciiPixel.jl

```diff
@@ -0,0 +1,39 @@
+module AsciiPixel
+
+using ImageBase: restrict
+using ImageCore
+using Crayons
+
+export ascii_encode
+
+include("colorant2ansi.jl")
+include("ascii_encode.jl")
+
+const colormode = Ref{TermColorDepth}(TermColor8bit())
+
+"""
+    set_colordepth(bit::Int)
+
+Sets the terminal color depth to the given argument.
+"""
+function set_colordepth(bit::Int)
+    if bit == 8
+        colormode[] = TermColor8bit()
+    elseif bit == 24
+        colormode[] = TermColor24bit()
+    else
+        error("Setting color depth to $bit-bit is not supported, valid mode are:
+          - 8bit (256 colors)
+          - 24bit")
+    end
+    colormode[]
+end
+
+set_colordepth(bit::AbstractString) = set_colordepth(parse(Int, replace(bit, r"[^0-9]"=>"")))
```

Comment:
This looks unnecessary to me; what's the advantage of `set_colordepth("8")` over `set_colordepth(8)`?

Reply:
All right, will remove `String` parsing.